patches/cloud-hypervisor/0001-build-chv-faster.patch

```diff
@@ -2,15 +2,12 @@ diff --git a/Cargo.toml b/Cargo.toml
 index 04b94ea9a..a2c8bd5e1 100644
 --- a/Cargo.toml
 +++ b/Cargo.toml
-@@ -18,10 +18,7 @@ version = "46.0.0"
- rust-version = "1.83.0"
-
+@@ -20,7 +20,7 @@ rust-version = "1.88.0"
  [profile.release]
--codegen-units = 1
--lto = true
+ codegen-units = 1
+ lto = true
 -opt-level = "s"
--strip = true
 +opt-level = 2
 
- [profile.profiling]
- debug = true
+ # Tradeof between performance and fast compilation times for local testing and
+ # development with frequent rebuilds.
```

Comment:
With https://github.com/cyberus-technology/libvirt-tests/pull/82 being merged, you should be able to remove that code again.